cockatrice/src/dlg_add_set.cpp

```diff
@@ -0,0 +1,37 @@
+#include "dlg_add_set.h"
+
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QVBoxLayout>
+
+DlgAddSet::DlgAddSet(QWidget *parent, bool success) : QDialog(parent) {
+    status = new QLabel(this);
+    restart = new QLabel(this);
+
+    if (success) {
+        setWindowTitle(tr("Success"));
+        status->setText(QString("Set added to Cockatrice."));
+        restart->setText(QString("You must restart to use your new set."));
```

Comment:
"You must restart **Cockatrice** to use *your* new set."
Otherwise some people might want to restart their PC... :)

I would prefer "the" over "your", too.